draft-ietf-add-ddr.md

```diff
@@ -451,11 +451,16 @@ authentication-domain-name (ADN).
 # Security Considerations
 
 Since clients can receive DNS SVCB answers over unencrypted DNS, on-path
-attackers can prevent successful discovery by dropping SVCB queries or answers.
+attackers can prevent successful discovery by dropping SVCB queries or answers,
+and thus prevent clients from switching to use encrypted DNS.
 Clients should be aware that it might not be possible to distinguish between
 resolvers that do not have any Designated Resolver and such an active attack.
 To limit the impact of discovery queries being dropped either maliciously or
 unintentionally, clients can re-send their SVCB queries periodically.
+{{Section 8.2 of !I-D.ietf-add-svcb-dns}} describes the downgrade attack,
+and explains that clients ought to switch to SVCB-reliant behavior once
+SVCB resolution does succeed. That is, encrypted DNS ought to be used if
+possible once an SVCB answer is received and the resolver can be validated.
```

Comment:
```suggestion

{{Section 8.2 of !I-D.ietf-add-svcb-dns}} describes a second downgrade attack,
and recommends that clients prevent it by switching to SVCB-reliant behavior once
SVCB resolution does succeed. For DDR, this means that once a client attempts
an encrypted DNS connection, it SHOULD NOT use unencrypted DNS until the
SVCB record expires.
```

Reply:
@bemasc how is this a second downgrade attack? The case of an attacker modifying network traffic to drop the SVCB records seems to be the same one?

Reply:
(Other than that, I'm fine with the proposed text)

Reply:
In SVCB-DNS, Section 8.1 is "Adversary on the query path", which is the preceding paragraph.  Section 8.2 is "Adversary on the transport path", which is the topic here.  These two paths are not necessarily coincident in DDR.

Reply:
Ah I see, how about:

```suggestion
{{Section 8.2 of !I-D.ietf-add-svcb-dns}} describes a second downgrade attack
where an attacker can block connections to the encrypted DNS server,
and recommends that clients prevent it by switching to SVCB-reliant behavior once
SVCB resolution does succeed. For DDR, this means that once a client attempts
an encrypted DNS connection, it SHOULD NOT use unencrypted DNS until the
SVCB record expires.
```

Reply:
Works for me.